insights/specs/default.py

```diff
@@ -699,6 +699,7 @@ def sap_sid(broker):
     yum_log = simple_file("/var/log/yum.log")
     yum_repolist = simple_command("/usr/bin/yum -C --noplugins repolist")
     yum_repos_d = glob_file("/etc/yum.repos.d/*.repo")
+    yum_updateinfo = simple_command("/usr/bin/yum -C updateinfo list")
```

Comment:
Once this information is collected it will be included in an archive and therefore you need to add a `simple_file` spec to the `specs/insights_archive.py` file.  Use the command `mangle "/usr/bin/yum -C --noplugins repolist"` to determine the filename to use in the spec.

Reply:
Added. Thanks